src/test/java/io/cdap/plugin/format/error/ErrorSchemaTest.java

```diff
@@ -0,0 +1,35 @@
+package io.cdap.plugin.format.error;
+
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import io.cdap.cdap.etl.api.InvalidEntry;
+import io.cdap.plugin.format.RecordWrapper;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ErrorSchemaTest {
+
+  @Test
+  public void testCreateErrorRecord() {
+    RecordWrapper wrapper = ErrorSchema.errorRecordWrapper("errorMessage", "exceptionClass", "tableName");
+
+    InvalidEntry<StructuredRecord> invalidEntry = wrapper.getInvalidEntry();
+    StructuredRecord invalidRecord = invalidEntry.getInvalidRecord();
+
+    Assert.assertEquals("errorMessage", invalidRecord.get(ErrorSchema.ERROR_MESSAGE));
+    Assert.assertEquals("exceptionClass", invalidRecord.get(ErrorSchema.EXCEPTION_CLASS_NAME));
+    Assert.assertEquals("tableName", invalidRecord.get(ErrorSchema.TABLE_NAME));
+  }
+
+  @Test
+  public void testCreateErrorRecordWithNullTable() {
+    RecordWrapper wrapper = ErrorSchema.errorRecordWrapper("errorMessage", "exceptionClass", null);
+
+    InvalidEntry<StructuredRecord> invalidEntry = wrapper.getInvalidEntry();
+    StructuredRecord invalidRecord = invalidEntry.getInvalidRecord();
+
+    Assert.assertEquals("errorMessage", invalidRecord.get(ErrorSchema.ERROR_MESSAGE));
+    Assert.assertEquals("exceptionClass", invalidRecord.get(ErrorSchema.EXCEPTION_CLASS_NAME));
+    Assert.assertNull(invalidRecord.get(ErrorSchema.TABLE_NAME));
+  }
+
+}
```

Comment:
missing copyright and newline at end of file. Supposed to be caught by checkstyle, this project probably doesn't have it setup?

Reply:
Oops, you are indeed correct. Neither my IDE or the project have the proper copyright config.